updated files to use the April 2023 cpu, added JDK 8 besides serverJRE 8 (#2614)
oracle/docker-images: OracleJava/README.md

# Oracle Java in Containers

> This repository contains sample container configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux with JDK images of JDK 20, 17, 11, 8 and for Server JRE 8.

Reviewer: ```suggestion
This repository contains sample container configurations to facilitate installation and environment setup for DevOps users. This project provides container images based on Oracle Linux for JDK versions 20, 17, 11 and 8 as well as Server JRE 8.
```
